### ISSUE TITLE
is "webpack": "^3.11.0" and "webpack-dev-server": "^2.11.2" installed locally?

### ISSUE THREAD

**Issue by BryanWilhite** [BryanWilhite/Songhay.Blog#29]
[issue 5](https://github.com/BryanWilhite/angular.io-index-app/issues/5) for `angular.io-index-app` records the discovery that `"webpack": "^3.11.0"` and `"webpack-dev-server": "^2.11.2"` are not installed locally

**Comment by BryanWilhite** [comment]
i ignorantly and optimistically installed newer versions:

```json
"webpack": "^4.16.4",
"webpack-dev-server": "^3.1.5"
```

some kind of `NodeTemplatePlugin.apply` error was my reward:

```console
An error occured during the build:
TypeError: Cannot read property 'thisCompilation' of undefined
    at NodeTemplatePlugin.apply (.\Git\Songhay.Blog\Songhay.Blog\ClientApp\node_modules\webpack\lib\node\NodeTemplatePlugin.js:19:18)
    at Compiler.apply (.\Git\Songhay.Blog\Songhay.Blog\ClientApp\node_modules\tapable\lib\Tapable.js:375:16)
    at Object.compileTemplate (.\Git\Songhay.Blog\Songhay.Blog\ClientApp\node_modules\html-webpack-plugin\lib\compiler.js:48:17)
    at Compiler.<anonymous> (.\Git\Songhay.Blog\Songhay.Blog\ClientApp\node_modules\html-webpack-plugin\index.js:47:40)
```

**Comment by BryanWilhite** [comment]
according to [an Angular CLI issue](https://github.com/angular/angular-cli/issues/9794), i should rollback:

![image](https://user-images.githubusercontent.com/5226627/43668758-67a190c0-9733-11e8-8399-3f0d5a52832e.png)

**Comment by BryanWilhite** [comment]
digging deeper into [the other Angular CLI issue](https://github.com/angular/angular-cli/issues/9793) it informs me that webpack is _inside_ the CLI?

![image](https://user-images.githubusercontent.com/5226627/43669209-3413b442-9736-11e8-9a01-6199d7f9eb53.png)

since the `angular.io-index-app` install installed webpack for me this possibility of having multiple webpack installs appears resolved (`"@angular/cli": "^1.7.4"`)

ran this locally:

```console
npm install webpack/3.11.0 webpack-dev-server/2.11.2
```
